File: Reverse.js
function reverseWordsInSentence(sentence) {
    // Split the sentence into words
    const words = sentence.split(' ');
  
    // Reverse each word and store them in a new array
    const reversedWords = words.map(word => {
      return word.split('').reverse().join('');
    });
  
    // Join the reversed words back into a sentence
    const reversedSentence = reversedWords.join(' ');
  
    return reversedSentence;
  }
  
  const inputSentence = "Hi Ajay, How are you..!";
  const reversedResult = reverseWordsInSentence(inputSentence);
  console.log(reversedResult);
  

File: Reverse.java
public class Reverse {
    public static void main(String[] args) {
        String str="Ajay";
        char ch[]=str.toCharArray();
        for(int i=ch.length-1;i>=0;i--){
            System.out.print(ch[i]);
        }
    }
}
